app/payloads/api/v1/medical_history_transformer.rb

```diff
@@ -0,0 +1,20 @@
+module Api::V1::MedicalHistoryTransformer
+  MEDICAL_HISTORY_QUESTIONS = [
+    :prior_heart_attack,
+    :prior_stroke,
+    :chronic_kidney_disease,
+    :receiving_treatment_for_hypertension,
+    :diabetes,
+    :diagnosed_with_hypertension
+  ].freeze
+
+  def self.medical_history_questions
+    MEDICAL_HISTORY_QUESTIONS
+  end
```

Comment:
This looks a bit weird. Can't we access module constants outside directly?

Reply:
I tried, but couldn't figure out a way to this.